docs/monitoring.md

```diff
@@ -176,6 +176,7 @@ Each instance can report to zero or more _sinks_. Sinks are contained in the
 * `JmxSink`: Registers metrics for viewing in a JMX console.
 * `MetricsServlet`: Adds a servlet within the existing Spark UI to serve metrics data as JSON data.
 * `GraphiteSink`: Sends metrics to a Graphite node.
+* `Slf4jSink`: Sends metrics to log4j as logging entries
```

Comment:
Couldn't it be sent from slf4j to another logging system too? it's possible to swap out the slf4j-log4j binding that Spark uses with a little work.